lib/galaxy/datatypes/dataproviders/column.py

```diff
@@ -360,7 +361,7 @@ class GoldenPathDataProvider(DictDataProvider):
     '''
     Class that defines a data provider for AGP datasets.
     '''
-    settings = {}
+    settings = {} # type: Dict[str, str]
```

Comment:
I guess you can directly use type hinting here :)
```suggestion
    settings: Dict[str, str] = {}
```